RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -627,15 +627,17 @@ boolean L12_getOutfit()
 	if(auto_warSide() == "fratboy" && possessOutfit("Filthy Hippy Disguise"))
 	{
 		autoOutfit("Filthy Hippy Disguise");
-		//r25671 this should be [Frat House In Disguise] but due to mafia issue it currently needs to be as below
+		//r25671 this should be [Frat House (Frat Disguise)] but due to mafia issue it currently needs to be as below
+		//r26631 "In Disguise" location names changed; this issue not checked
 		return autoAdv($location[Frat House]);
 	}
 	
 	// if outfit could not be pulled and have a [Frat Boy Ensemble] outfit then wear it and adventure in Hippy Camp to get war outfit
 	if(auto_warSide() == "hippy" && possessOutfit("Frat Boy Ensemble"))
 	{
 		autoOutfit("Frat Boy Ensemble");
-		//r25671 this should be [Hippy Camp In Disguise] but due to mafia issue it currently needs to be as below
+		//r25671 this should be [Hippy Camp (Hippy Disguise)] but due to mafia issue it currently needs to be as below
+		//r26631 "In Disguise" location names changed; this issue not checked
 		return autoAdv($location[Hippy Camp]);
```

Comment:
`$location[Hippy Camp (Hippy Disguise)]` correctly returns Hippy Camp (Hippy Disguise) now, so this can be updated to the desired location now.

```suggestion
		return autoAdv($location[Hippy Camp (Hippy Disguise)]);
```

Reply:
only tested with frat outfit but assume this would also give 

    You can't get to that area.